frontend/kesaseteli/employer/src/utils/localstorage.utils.ts

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable scanjs-rules/identifier_localStorage */
+export const clearLocalStorage = (keyPrefix?: string): void => {
```

Comment:
localstorage is cleared when logging out or when sending application. Currently only states of accordions are saved into it.